src/main/java/com/guysfromusa/carsgame/services/CarService.java

```diff
@@ -117,4 +116,11 @@ public CarEntity addCarToGame(String carName, String gameName, Point startingPoi
         return carRepository.save(car);
     }
 
+    public CarEntity repairCar(String name) {
+        CarEntity car = carRepository.findByName(name)
+                .orElseThrow(() -> new EntityNotFoundException("Car with name " + name + " does not exist"));
+
+        car.setCrashed(false);
+        return carRepository.save(car);
```

Comment:
if the entity modification is in transaction then we don't need to perform save on repository

Reply:
this means you can skip last line (if you have repo test ;))